README.md

```diff
@@ -1 +1,102 @@
-# Global-Leadership-Summit
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👤 Author](#author)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Global Leadership Summit] <a name="about-project"></a>
+
+> This project was created to design a website for an upcoming event which assume to be hosted on United States.
+>Its is Microverse Module 1 Capstone Project to test or skills on all that we have built in the past.
+> It is built with flexbox, css grid,  media query for responsive web design and JavaScript Dom manipulation for menubar.
+>It have two pages, the Home page and the about page
+
+**Global Leadership Summit]** is a webpage with menu bar, hero section, featured speakers section, about section and About Page.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The tech stack used are html, css and js.
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Video Presentation link <a name="Video Presentation link"></a>
+- [Live Demo Link](https://www.loom.com/share/dcd06582bc6540608b4ae446cab6718d/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Deployment
```

Comment:
Great job @chibunduonyeje1 🤝

- [ ] As stated in the project requirements, kindly deploy your project using GitHub pages and add the deployed link to your readme file.